dubbo-serialization/dubbo-serialization-test/src/test/java/org/apache/dubbo/common/serialize/fst/FstObjectOutputTest.java

```diff
@@ -41,6 +42,11 @@ public void setUp() {
         this.fstObjectOutput = new FstObjectOutput(byteArrayOutputStream);
     }
 
+    @After
+    public void tearDown() throws IOException {
+        new FstObjectInput(new ByteArrayInputStream(new byte[]{0}));
```

Comment:
why a `new FstObjectInput()` is necessary in tearDown method?

Reply:
not sure whether `org.nustaq.serialization.FSTConfiguration#clearGlobalCaches` and `org.nustaq.serialization.FSTConfiguration#clearCaches` do the tricks.

Reply:
Thank you for the suggestions. Unfortunately, calling neither of the two methods helped here, the command still fails when using these two methods in the tearDown. It seems there is still some other state that these clear methods do not help reset.